From a9f6a55bdb29a2bebc96a68ab53077906c25a9df Mon Sep 17 00:00:00 2001
From: Yegor Yefremov <yegorslists@googlemail.com>
Date: Wed, 22 Jan 2014 15:04:42 +0100
Subject: [PATCH] Fix compiling on Buildroot

Buildroot always specifies -D_LARGEFILE_SOURCE, -D_LARGEFILE64_SOURCE,
-D_FILE_OFFSET_BITS=64, so define them only if they are not already
defined.

Signed-off-by: Yegor Yefremov <yegorslists@googlemail.com>
---
 src/fmacros.h |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/src/fmacros.h b/src/fmacros.h
index fa37948..059dfeb 100644
--- a/src/fmacros.h
+++ b/src/fmacros.h
@@ -14,7 +14,11 @@
 #define _XOPEN_SOURCE
 #endif

+#ifndef _LARGEFILE_SOURCE
 #define _LARGEFILE_SOURCE
+#endif
+#ifndef _FILE_OFFSET_BITS
 #define _FILE_OFFSET_BITS 64
+#endif

 #endif
--
1.7.7
